metabase/metabase
# fix waterfall series is missing on enormous datasets

Closes https://github.com/metabase/metabase/issues/39411

### Description

Candlestick series hides elements when they become too small, so this PR replaces it with `custom` series for waterfall charts.

### How to verify

Create a waterfall chart with a small enough bar width like Orders Count by Created At: Day and ensure it shows the waterfall series.

### Demo

<img width="1702" alt="Screenshot 2024-03-05 at 9 54 48 PM" src="https://github.com/metabase/metabase/assets/14301985/600e63da-d48a-40c3-a513-69714291c6ef">

### Checklist
